src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -197,35 +199,115 @@ def __call__(self, input_ids, token_type_ids, position_ids, attention_mask, dete
         hidden_states = inputs_embeds + jnp.broadcast_to(position_embeds, inputs_embeds.shape) + token_type_embeddings
 
         # Layer Norm
-        hidden_states = self.layer_norm(hidden_states)
+        hidden_states = self.LayerNorm(hidden_states)
         hidden_states = self.dropout(hidden_states, deterministic=deterministic)
         return hidden_states
 
 
-class FlaxBertAttention(nn.Module):
+class FlaxBertSelfAttention(nn.Module):
     config: BertConfig
     dtype: jnp.dtype = jnp.float32  # the dtype of the computation
 
     def setup(self):
-        self.self_attention = nn.attention.SelfAttention(
```

Comment:
`flax.linen.SelfAttention` is removed and query, key and value weights are created in the same way we do it for PyTorch